### PR TITLE
[BH-1270] Fix idle timeout in applications

### PR DESCRIPTION
**Pull request by tigransoghbatyan-digica**
Implement correct idle timeout handling in applications

### CODE REVIEW

File: products/BellHybrid/apps/Application.cpp [hunk 1]
```diff
@@ -86,6 +90,18 @@ namespace app
...
     }
 
+    sys::MessagePointer Application::handleInputEvent(sys::Message *msgl)
+    {
+        AppInputEventMessage *msg  = reinterpret_cast<AppInputEventMessage *>(msgl);
+        gui::InputEvent inputEvent = msg->getEvent();
+        if (inputEvent.isShortRelease(gui::KeyCode::KEY_RIGHT)) {
+            std::unique_ptr<bell_settings::FrontlightModel> frontlightModel;
+            // frontlightModel.get()->setBrightness(5);
+            LOG_INFO("-----------+++++++=----------");
```
Comment (mjan-dev) [inline products/BellHybrid/apps/Application.cpp:100]: Please clean the logs.

File: products/BellHybrid/apps/Application.cpp [hunk 2]
```diff
@@ -143,4 +161,15 @@ namespace app
 
     void Application::updateStatuses(gui::AppWindow *window) const
     {}
+
+    void Application::reasumeIdleTimer()
```
Comment (mjan-dev) [inline products/BellHybrid/apps/Application.cpp:165]:
```suggestion
    void Application::resumeIdleTimer()
```

**Review by mjan-dev — COMMENTED** [review]
Please switch to stable.

File: products/BellHybrid/apps/Application.cpp (hunk 2 above)
Comment (tigransoghbatyan-digica) [reply products/BellHybrid/apps/Application.cpp]: done

**Review by mjan-dev — APPROVED** [review]

**Pull request merged**